presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformUncorrelatedScalarToJoin.java

```diff
@@ -18,16 +18,23 @@
 import com.facebook.presto.sql.planner.PlanNodeIdAllocator;
```

Comment:
It looks like this code is no longer needed. I remember I had to add it at some point where I had some test failures otherwise, but today I removed this code and no failure occurred.

I am going to remove this commit as it looks to be a dead code.